charters/DataStore/charter.md

```diff
@@ -0,0 +1,72 @@
+
+# Data Store
+
+
+## Description
+The Data Store is a scientific data sharing/publishing/distribution framework, providing file/bundle management on multiple clouds at petabyte-scale. It defines public APIs for storage, retrieval, and subscription to events that functions transparently across multiple cloud systems such as AWS and GCP.
+
+## Objective
+The objective of the Data Store group is to deliver a versioned immutable object based data repository that is highly available and scalable. Data will be replicated to at least two commercial clouds (Amazon and Google). Data will be accessible through a variety of programatic interfaces as well as a command line interface.
+
+## In-scope
```

Comment:
Another in-scope item:  DevOps

Reply:
I initially did not include DevOps as I perceived the charters as a list of tasks with definitive endpoints.  DevOps will be on-going. Discussions had since then have changed my view and I will add a section to spell out responsibility for DevOps.